eth/vm/base.py

```diff
@@ -640,7 +639,7 @@ def set_block_transactions(self,
     #
     def _assign_block_rewards(self, block: BaseBlock) -> None:
         block_reward = self.get_block_reward() + (
-            len(block.uncles) * self.get_nephew_reward()
+                len(block.uncles) * self.get_nephew_reward()
```

Comment:
This indentation change looks like it should be reverted.

Reply:
this was solved in the 9bd96bb commit